.github/workflows/release-pr.yaml

```diff
@@ -74,32 +74,32 @@ jobs:
         run: ./scripts/test-env.sh
 
       - name: Run chart-testing (install)
-        run: ct install --charts charts/${{ matrix.chart-name}}
+        run: ct install --target-branch main --charts charts/${{ matrix.chart-name}}
 
   integration-test:
     timeout-minutes: 30
     runs-on: ubuntu-latest
     env:
       # Specify this here because these tests rely on ktf to run kind for cluster creation.
-      KIND_VERSION: v0.20.0
-      GATEWAY_API_VERSION: v0.8.1
+      KIND_VERSION: v0.22.0
+      GATEWAY_API_VERSION: v1.0.0
     strategy:
       matrix:
         kubernetes-version:
-          - "1.23.13"
-          - "1.24.7"
-          - "1.25.9"
-          - "1.26.4"
-          - "1.27.1"
+          - "1.23.17"
+          - "1.24.17"
```

Comment:
```suggestion
```
I think we could drop the two oldest versions after adding the two newer ones (that would reflect the approach we follow in KIC).

Reply:
Those are still present in e.g. https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/version-compatibility/#general (at least for older versions which we also tests in the integration tests suite). I'd leave them until those versions go out of support